src/account-info/AccountDetails.js

```diff
@@ -60,12 +66,14 @@ class AccountDetails extends PureComponent<Props> {
       }
     }
 
+    const botOwner = user.is_bot ? usersById.get(user.bot_owner_id || 0) : undefined;
+
     return (
-      <ComponentList outerSpacing spacing={12} itemStyle={componentStyles.componentListItem}>
-        <View>
+      <ComponentList outerSpacing spacing={12}>
+        <View style={componentStyles.statusWrapper}>
           <UserAvatar avatarUrl={getAvatarFromUser(user, realm, AVATAR_SIZE)} size={AVATAR_SIZE} />
         </View>
-        <View style={componentStyles.statusWrapper}>
+        <View style={[componentStyles.statusWrapper, componentStyles.componentListItem]}>
```

Comment:
These style changes also need explanation; it's not clear what their purpose is.